### ISSUE TITLE
Don't allow Ctrl+C to kill the outermost command processor

### ISSUE THREAD

**Issue by GoogleCodeExporter** [rheostat2718/conemu-maximus5#900]
```
I have the habit of pushing "Ctrl+C" over and over again without looking if I 
want to terminate everything and get back to the "base console" prompt.

With cmd.exe - this is fine because pushing "Ctrl+C" on an outer instance will 
do nothing.

However, with ConEmu, Ctrl+C also prompts for and can kill the outermost 
cmd.exe process. This seems awkward to me. If I wanted to close the window - I 
would use the hotkey Alt+F4. Also if cmd.exe is run inside of cmd.exe - then I 
want Ctrl+C (and its confirmation) to only show if I am in a nested cmd.exe 
instance.

In short, Ctrl+C should not have the ability to be able to terminate the 
window. Ctrl+C should only be able to terminate subprocesses running INSIDE the 
window.
```

Original issue reported on code.google.com by `robertco...@yahoo.com` on 21 Jan 2013 at 7:29
